front/src/modules/timetableItem/components/Timetable/RoundTrips/RoundTripsModalColumn.tsx

```diff
@@ -5,17 +5,17 @@ import { useTranslation } from 'react-i18next';
 import type { PairingItem } from '../types';
 import RoundTripsModalCard from './RoundTripsModalCard';
 
-type RoundTripsModalColumnProps =
+type RoundTripsModalColumnProps = { isLoading: boolean } & (
```

Comment:
Since placeholders are displayed with `pairingItems.length === 0`, do we need this extra field?